priv/templates/nimble_template/assets/.stylelintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": ["stylelint-config-sass-guidelines", "stylelint-config-property-sort-order-smacss"],
```

Comment:
How about

```suggestion
  "extends": [
    "stylelint-config-sass-guidelines", 
    "stylelint-config-property-sort-order-smacss"
  ],
```

Reply:
yeah, I have updated it in https://github.com/nimblehq/elixir-templates/pull/144/commits/40ded4d770d9f74aa6699fe458fed4922d7b8bac Ros 👍